### PR TITLE
manually add elevatezoom 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/elevateweb/elevatezoom
**Watch 56
  Star 698
 Fork 282**
@PeterDaveHello  could u help me check this PR for #5961  ? thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update) https://github.com/elevateweb/elevatezoom/issues/120
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

**since the author doesn't act on github for a long time
i add 3 released version without git auto-update config
because it's minified file name include version number,
i remove all minified file version number in their name
i have opened a issue to ask if the author can remove
the version number from the file name [here](https://github.com/elevateweb/elevatezoom/issues/140)**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Why to add image?

**Comment by maruilian11** [comment]
@PeterDaveHello sorry, we don't need them.

**Comment by PeterDaveHello** [comment]
Should also use something like `jquery.elevateZoom*.js` as auto-update config for future releases, even it can't match our requirement 100%

File: ajax/libs/elevatezoom/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "effects",
+    "elements",
+    "zooming"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/elevateweb/elevatezoom.git",
+    "basePath": "",
+    "files": [
+      "jquery.elevatezoom*.+(js|map)"
```
Comment (maruilian11) [inline ajax/libs/elevatezoom/package.json]: @PeterDaveHello is that ok?

Comment (PeterDaveHello) [inline ajax/libs/elevatezoom/package.json]: And `jquery.elevateZoom` ...

File: ajax/libs/elevatezoom/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "effects",
+    "elements",
+    "zooming"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/elevateweb/elevatezoom.git",
+    "basePath": "",
+    "files": [
+      "jquery.+(elevatezoom|elevateZoom)*.+(js|map)"
```
Comment (maruilian11) [inline ajax/libs/elevatezoom/package.json]: @PeterDaveHello is that ok?

**Comment by PeterDaveHello** [comment]
Try to add more as they are versioned but not tagged, examples:

3.0.8:
https://github.com/elevateweb/elevatezoom/tree/81c2cc4946f6bebc33c0d8e804b046b36fa1bc61

3.0.7:
https://github.com/elevateweb/elevatezoom/tree/ebb1f17303ac9ab6f936e481463076755904ef43

3.0.6
https://github.com/elevateweb/elevatezoom/tree/a332e67c1a287bd300b07de3489225c62b172220

3.0.5
https://github.com/elevateweb/elevatezoom/tree/1dd92144e1670f754a75b2381316e10c18d4dad8

3.0.4
https://github.com/elevateweb/elevatezoom/tree/f4a41505fbd4697b09016c87370db4ea1df8eeef

3.0.3
https://github.com/elevateweb/elevatezoom/tree/ce01ceab11da2f3a46e8176ea30b43dbc8405a1f

3.0.2
https://github.com/elevateweb/elevatezoom/tree/ff1444d8d036933538e9344b95e4cbc8941823be

3.0.1
https://github.com/elevateweb/elevatezoom/tree/279132a435d5facbcd1c4516ad92ac48c058010d

3.0.0
https://github.com/elevateweb/elevatezoom/tree/f14ebf0e53ce1e2d62c44214b3ff12b2e899c985

**Comment by maruilian11** [comment]
@PeterDaveHello could you tell me how do you find those version not tagged? do you grep the file name in the changed files in log?

**Comment by PeterDaveHello** [comment]
@maruilian11 Just trace the log.

**Comment by maruilian11** [comment]
@PeterDaveHello i have added v3.0.0 ~ v3.0.8.

**Comment by PeterDaveHello** [comment]
@maruilian11 there are not only these missing versions ... like v2.6.0

**Comment by maruilian11** [comment]
2.1.0
https://github.com/elevateweb/elevatezoom/tree/40cba6732614199865c07c5ce6660c4f1cddd658
2.2.0
https://github.com/elevateweb/elevatezoom/tree/b0ea5699fd36ffeb02201fd0c02b96f442032465
2.2.3
https://github.com/elevateweb/elevatezoom/tree/f6bc619b16ae3d136de524a991dc71dd7e912849
2.3.0
https://github.com/elevateweb/elevatezoom/tree/963e2ed8d0eacdf0a9dd654b8d362b1f400abe1a
2.4.0
https://github.com/elevateweb/elevatezoom/tree/514c54d160e5dc852845992046c3f901b0170b4e
2.5.0
https://github.com/elevateweb/elevatezoom/tree/16e08f5d0e315e9ae9df46386b60c4d2e0835c9e
2.5.2
https://github.com/elevateweb/elevatezoom/tree/9c740e7738a4fd83b906584c909fdb3cc7577655
2.5.3
https://github.com/elevateweb/elevatezoom/tree/0877cba18a276933841e9a67e52ddbfd626b7e4e
2.5.4
https://github.com/elevateweb/elevatezoom/tree/76ae3b9e7face70a4d9c20497688c72890ef3438
2.5.5
https://github.com/elevateweb/elevatezoom/tree/d14299094470d40d213fc0dba6a59e7376af58d9
2.5.6
https://github.com/elevateweb/elevatezoom/tree/b65a8f6f61440d5e792c50b903c4b56e80e05d26
2.6.0
https://github.com/elevateweb/elevatezoom/tree/8a6652a51fd373bf8318655cf9bdc326009df7f1

**Comment by PeterDaveHello** [comment]
Why your 2.2.0 looks like v2.1.2?
https://github.com/elevateweb/elevatezoom/tree/b0ea5699fd36ffeb02201fd0c02b96f442032465

**Comment by maruilian11** [comment]
https://github.com/elevateweb/elevatezoom/commit/b0ea5699fd36ffeb02201fd0c02b96f442032465
i think the author modified the `jquery.elevatezoom.js` to v2.2.0 but he forgot to minified it, i will minified the file again to replace `jquery.elevateZoom-2.1.2.min.js`

**Comment by maruilian11** [comment]
oh i already did it...
i used web-minify-helper to minify `jquery.elevatezoom.js` v2.2.0.
you can see `jquery.elevatezoom.min.js` and `jquery.elevatezoom.min.js.map`.

**Comment by PeterDaveHello** [comment]
So what's the commit of v2.1.2?

**Comment by maruilian11** [comment]
this one https://github.com/elevateweb/elevatezoom/commit/d5808a3ab4cc1a5a47e7e93c4243d081ea679359
and i got v2.1.2 from https://github.com/elevateweb/elevatezoom/releases

**Comment by PeterDaveHello** [comment]
So update your comment https://github.com/cdnjs/cdnjs/pull/8039#issuecomment-222719252 for a full and clear list

**Comment by maruilian11** [comment]
i have updated the list.

**Comment by PeterDaveHello** [comment]
Can you use a single and consistent format for review? You leave https://github.com/elevateweb/elevatezoom/releases for some versinos, why not just the commit as the others? If you want some others to review your work, don't make things so complex and high overhead.

**Comment by maruilian11** [comment]
@PeterDaveHello sorry for that, i have updated the list.

**Comment by PeterDaveHello** [comment]
Let @Amomo review this PR.

**Comment by Amomo** [comment]
Thanks for both of you finding out these commits... 😭 
I have checked files in this PR with the original repo.

@maruilian11 I found there are two parts wrong.
Your `2.1.1/jquery.elevateZoom.min.js` is v2.1.0. (written in its comment field)
See this commit https://github.com/elevateweb/elevatezoom/commit/d5808a3ab4cc1a5a47e7e93c4243d081ea679359 , the author didn't generate a minified version for v2.1.1. Please generate one for this version.

And, your `2.1.0/jquery.elevatezoom.js` is v2.1.1.(also written in its comment field)
you can use https://github.com/elevateweb/elevatezoom/commit/f2cd71aba6335ebe67697965b2a3217adf49c489 as v2.1.0.

**Comment by maruilian11** [comment]
@Amomo thank you for your review, i have added v2.1.0 from https://github.com/elevateweb/elevatezoom/commit/f2cd71aba6335ebe67697965b2a3217adf49c489 as v2.1.0.
and reminify the js of v2.1.1.

**Comment by maruilian11** [comment]
ping @Amomo

**Comment by PeterDaveHello** [comment]
Do not manually generate the map file in this case, which makes inconsistency in this lib.

**Comment by maruilian11** [comment]
@Amomo i removed all `map` files, could you help me check this PR? thanks!

File: ajax/libs/elevatezoom/2.1.1/jquery.elevatezoom.min.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+//# sourceMappingURL=jquery.elevatezoom.min.js.map
```
Comment (PeterDaveHello) [inline ajax/libs/elevatezoom/2.1.1/jquery.elevatezoom.min.js]: So why this js file point to a map file?

**Comment by PeterDaveHello** [comment]
There are some `jquery.elevateZoom.min.js`s and `jquery.elevatezoom.min.js`s, shuold just use `jquery.elevatezoom.min.js`.

**Comment by PeterDaveHello** [comment]
Bad commit message again, and again.

**Comment by maruilian11** [comment]
@PeterDaveHello i removed the lines pointing to map file, renamed `jquery.elevateZoom.min.js`s and made commit message a sentence.

**Comment by PeterDaveHello** [comment]
You DIDN'T FIX the FILENAME PROBLEM :bomb:

**Comment by maruilian11** [comment]
@PeterDaveHello hmmm i really did the steps...
>>>
@PeterDaveHello i removed the lines pointing to map file, renamed `jquery.elevateZoom.min.js`s and made commit message a sentence. 

but i didn't check it after rebased with our latest master, i was in a hurry. 
maybe some problem happened during the rebasing, i will check if the commit match what i have done before tagging you next time. sorry...

**Comment by PeterDaveHello** [comment]
I have no idea what's the reason why to quote these sentence, another mistake?

**Comment by PeterDaveHello** [comment]
Don't you need to mention the reason why you manually add those versions?

**Comment by maruilian11** [comment]
@PeterDaveHello is that ok?

**Pull request merged**